script/assert_max_wasm_size.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echoerr() { echo "$@" 1>&2; }
+
+current_size=$(cat file_size)
+current_size="${current_size:0:$((${#current_size} - 1))}"
```

Comment:
What does this line do? Bash is magic, could you please refactor it to more variables please? By the way, I find bash super cool for 1-line scripts, but even with 10-lines scripts it starts to be unreadable. Maybe we should create bigger (bigger means more than 3 lines :P) scripts in python or Rust instead? https://github.com/DanielKeep/cargo-script

Reply:
It slices out the trailing M in 123.4M. I think this should be clear enough:

```
current_size=$(cat file_size)
current_size_len=$((${#current_size} - 1))
current_size="${current_size:0:current_size_len}"
```

But I will keep in mind cargo-script for future cases.